plugin-compass/src/test/java/com/mapbox/maps/plugin/compass/CompassViewPluginTest.kt

```diff
@@ -359,8 +351,7 @@ class CompassViewPluginTest {
       0.0,
       0.0,
       -0.23,
-      null,
-      null
+      arrayOf(0.0, 0.0, 0.0, 0.0)
```

Comment:
could we leverage a default value instead?

Reply:
Sure